cmd/finalize/finalize.go

```diff
@@ -33,6 +36,8 @@ func main() {
 	defer mq.Connection.Close()
 	defer db.Close()
 
+	ingestAccession := gojsonschema.NewReferenceLoader("https://raw.githubusercontent.com/EGA-archive/LocalEGA/master/ingestion/schemas/ingestion-accession.json")
```

Comment:
I would create a local fallback to a schema and log that the schema there does not exist to avoid URL changing without notice

Reply:
Not only URL: content can be changed as well.

Reply:
true, so maybe a separate module for loading schemas and adding a fallback option inside e.g. erroring with specific message like schema content changed, url not accessible etc. ?

Reply:
or that is too complicated for this version ?

Reply:
I think it is. I would personally just dump their current schemas and would use local files in the validation. Apart from that, we can set up a completely separate cron-based job (maybe on Travis) to daily check that their schemas are untouched, so that we get notified if URLs/content changes.